python/test/test_polars.py

```diff
@@ -681,16 +680,16 @@ def test_pickle_bomb():
     pl.LazyFrame.deserialize(io.BytesIO(ser_lf))
 
     # OpenDP explicitly rejects any pickled data it finds
-    err_msg_re = re.escape("OpenDP does not allow pickled keyword arguments as they may enable remote code execution.")
-    with pytest.raises(dp.OpenDPException, match=err_msg_re):
+    err_msg = "OpenDP does not allow pickled keyword arguments as they may enable remote code execution."
+    with pytest.raises(dp.OpenDPException, match=err_msg):
```

Comment:
I would kind of like to keep `re` in the name. I've forgotten that it's not a string match and been a little confused debugging a test.